### PR TITLE
[server-wallet] Pass channelId to serializeMessage

### PR DESCRIPTION
**Pull request by geoknee**
It's an optional parameter which means the function can skip computation of the `channelId` in the case it is already known.

### CODE REVIEW

**Comment by geoknee** [comment]
Seeing `makePayment` take [2643 ms](https://app.circleci.com/pipelines/github/statechannels/statechannels/9101/workflows/e9a74dd5-afbc-417e-a4ff-79f636e907a7/jobs/43597) in the stress test 😕

**Comment by snario** [comment]
I re-ran it and saw [2403 ms](https://app.circleci.com/pipelines/github/statechannels/statechannels/9101/workflows/471b1d1c-e710-4e38-88d1-a517e9593d91/jobs/43657/parallel-runs/0/steps/0-105)

File: packages/server-wallet/src/wallet/__test__/integration/push-message.test.ts [hunk 1]
```diff
@@ -87,9 +87,10 @@ describe('channel results', () => {
     const createChannel = createChannelFromState(signedStates[0]);
     await wallet.pushMessage({objectives: [createChannel]});
 
-    await expectResults(wallet.pushMessage({signedStates: signedStates.map(serializeState)}), [
-      {turnNum: zero, status: 'proposed'},
-    ]);
+    await expectResults(
+      wallet.pushMessage({signedStates: signedStates.map(ss => serializeState(ss))}),
```
Comment (tomclose) [inline packages/server-wallet/src/wallet/__test__/integration/push-message.test.ts:91]: Was this change deliberate, or is this prettier?

**Review by tomclose — APPROVED** [review]

File: packages/server-wallet/src/wallet/__test__/integration/push-message.test.ts (hunk 1 above)
Comment (geoknee) [reply packages/server-wallet/src/wallet/__test__/integration/push-message.test.ts:91]: I think deliberate, but I can't remember exactly why I did it
